app/src/main/java/com/tr4n/moviedb/ui/detail/TabAboutMovieFragment.kt

```diff
@@ -2,27 +2,28 @@ package com.tr4n.moviedb.ui.detail
 
 import com.tr4n.moviedb.R
 import com.tr4n.moviedb.base.BaseFragment
+import kotlinx.android.synthetic.main.fragment_tab_about_movie.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class TabAboutMovieFragment : BaseFragment(R.layout.fragment_tab_about_movie) {
-
-    companion object {
-        fun newInstance() = TabAboutMovieFragment()
-    }
+class TabAboutMovieFragment(private val movieId: Long) : BaseFragment(R.layout.fragment_tab_about_movie) {
```

Comment:
pass qua bundle, không truyền qua constructor 
```suggestion
class TabAboutMovieFragment(private val movieId: Long) : BaseFragment(R.layout.fragment_tab_about_movie) {
```